src/setup-node.ts

```diff
@@ -17,6 +19,15 @@ async function run() {
       await installer.getNode(version);
     }
 
+    // Output version of node and npm that are being used
+    const nodePath = await io.which('node');
+    const nodeVersion = cp.execSync(`${nodePath} --version`);
```

Comment:
Doesn't execSync return a buffer of bytes that you need toString() (safely)?  See https://github.com/actions/setup-go/blob/master/src/main.ts#L56

Reply:
I thought template strings (ex `${nodePath} --version`) already called `.toString()`

Reply:
Ahhh - yeah.  I think you're right.  Just e2e test